planetiler-core/src/main/java/com/onthegomap/planetiler/config/PlanetilerConfig.java

```diff
@@ -99,6 +101,8 @@ public static PlanetilerConfig from(Arguments arguments) {
       arguments.getDuration("loginterval", "time between logs", "10s"),
       arguments.getInteger("minzoom", "minimum zoom level", MIN_MINZOOM),
       arguments.getInteger("maxzoom", "maximum zoom level (limit 14)", MAX_MAXZOOM),
+      arguments.getInteger("minzoomForRendering", "minimum zoom level", MIN_MINZOOM),
```

Comment:
I don't think min zoom  for rendering is really necessary - I see it's only used in CacheByZoom which just uses the thresholds to set a size for the value array ahead of time - so just using min zoom would be fine.